modules/ROOT/pages/app-to-app-requests.adoc

```diff
@@ -0,0 +1,23 @@
+= Access an Application in the Same Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In an Anypoint Runtime Fabric cluster, every application replica exposes an endpoint inside the cluster. When you enable inbound traffic, that endpoint is exposed externally. Refer to xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric] for additional information about servicing external requests and configuring the load balancer. 
+
+To send a request from one application to another application inside the cluster, use the application name and environment ID. 
+
+Every application has a DNS record in the cluster internal domain. For example, the DNS record for the application `hello-world` in the  `2abc2ce6-cce6-463b-afaa-a1979e3xxxxx` environment is:
+```
+hello-world.2abc2ce6-cce6-463b-afaa-a1979e3xxxxx.svc.cluster.local
+```
+Because application replicas first search DNS records in the same domain before searching the parent domain, you need only the application name and environment ID to send a request to another application in the same cluster. If both applications are in the same namespace, you need only the application name:
```

Comment:
So...requests are made first internally and then externally by default?
Does "same domain" = "cluster"? Does "parent domain" = external to the cluster?

Again, I think I am getting confused by not understanding how the different terms relate to one thing or different things. It would be clearer to stick with internal terms, and clarify cluster, parent domain, child domain, environment ID, and application namespace and name.